src/makeCalculator.js

```diff
@@ -36,8 +36,44 @@
  *
  * @return {object}
  */
-function makeCalculator() {
-  // write code here
+function makeCalculator(callback, value) {
+  const calculator = {
+    result: 0,
+
+    add(prevValue, number) {
+      return prevValue + number;
+    },
+
+    subtract(prevValue, number) {
+      return prevValue - number;
+    },
+
+    divide(prevValue, number) {
+      if (number === 0) {
+        throw new Error('Dividing by zero! We keeping previous result');
+      } else {
+        return prevValue / number;
+      }
+    },
+
+    multiply(prevValue, number) {
+      return prevValue * number;
+    },
+
+    reset() {
+      this.result = 0;
```

Comment:
this `0`  you can keep in const as a start value

don't forget to use it in `result` as well